### ISSUE TITLE
Installed failed - no assets folder

### ISSUE THREAD

**Issue by CarsonF** [bolt/labels#1]
```
[command/00:32:12] > composer require 'bolt/labels' 'dev-master' -d "/Users/Carson/Repos/Godlife/sites/godlife.com/app/extensions" -n --no-ansi
[error/00:32:14] ./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
- Installing bolt/labels (dev-master 2612ddf)
Cloning 2612ddf2a113a05953f5b6286fe61046bf928cac

Script Bolt\Composer\ExtensionInstaller::handle handling the post-package-install event terminated with an exception



[UnexpectedValueException] 
RecursiveDirectoryIterator::__construct(/Users/Carson/Repos/Godlife/sites/godlife.com/app/extensions/vendor/bolt/labels/assets): failed to open dir: No such file or directory 
```

**Comment by tobias2k** [comment]
I'll look into it.

On Fri, Jan 09, 2015 at 04:35:22PM -0800, Carson Full wrote:

> [command/00:32:12] > composer require 'bolt/labels' 'dev-master' -d "/Users/Carson/Repos/Godlife/sites/godlife.com/app/extensions" -n --no-ansi
> [error/00:32:14] ./composer.json has been updated
> Loading composer repositories with package information
> Updating dependencies (including require-dev)
> - Installing bolt/labels (dev-master 2612ddf)
>   Cloning 2612ddf2a113a05953f5b6286fe61046bf928cac
> 
> Script Bolt\Composer\ExtensionInstaller::handle handling the post-package-install event terminated with an exception
> 
> [UnexpectedValueException] 
> RecursiveDirectoryIterator::__construct(/Users/Carson/Repos/Godlife/sites/godlife.com/app/extensions/vendor/bolt/labels/assets): failed to open dir: No such file or directory 
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/bolt/labels/issues/1

## 

Tobias Dammers - tobias@twokings.nl - 070-3457628 - www.twokings.nl
Maandag t/m donderdag van 9.00 tot 17.30
Voor dringende vragen, mail naar support@twokings.nl

**Comment by tobias2k** [comment]
Looks like we accidentally kept the reference to /assets in the composer file. I fixed that and released the fixed version as 1.0.2; it installs cleanly on a fresh test setup, so I hope it works for you now.
